tarantool_test.go

```diff
@@ -910,7 +910,7 @@ func TestClient(t *testing.T) {
 	}
 	//resp, err = conn.Insert(spaceNo, []interface{}{uint(1), "hello", "world"})
 	resp, err = conn.Insert(spaceNo, &Tuple{Id: 1, Msg: "hello", Name: "world"})
-	if tntErr, ok := err.(Error); !ok || tntErr.Code != ErrTupleFound {
```

Comment:
I think that having error codes exposed sounds rather useful for an application developer. Do we plan to include them in new IPROTO constants package?

Reply:
It would be implemented in IPROTO constants package:

https://github.com/tarantool/go-tarantool/issues/267#issuecomment-1479541164